internal/otelcollector/config/metric/gateway/processors.go

```diff
@@ -149,6 +152,42 @@ func makeFilterByNamespaceConfig(namespaceSelector *telemetryv1alpha1.MetricPipe
 	}
 }
 
+// Drop the metrics scraped by k8s cluster, except for the pod and container metrics
+// Complete list of the metrics is here: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/k8sclusterreceiver/documentation.md
+func makeK8sClusterDropMetrics() *FilterProcessor {
+	metricNames := []string{
+		"^k8s.deployment.*",
+		"^k8s.cronjob.*",
+		"^k8s.daemonset.*",
+		"^k8s.hpa.*",
+		"^k8s.job.*",
+		"^k8s.replicaset.*",
+		"^k8s.replication_controller.*",
```

Comment:
maybe we should also disable the `k8s.replication_controller` metrics in the receiver instead of dropping them in the processor? since we won't have any selectors for these metrics